src/d3a/d3a_core/live_events.py

```diff
@@ -55,19 +55,21 @@ def apply(self, area):
         if area_type is not None and area_type != "Area":
             if area.strategy is None:
                 return False
+            reactivate = False
             if area_type == "MarketMaker":
                 area.strategy = MarketMakerStrategy(**self.area_params)
+                reactivate = True
             elif area_type == "InfiniteBus":
                 # TODO: After hack to move this parameter casting at the web side
                 self.area_params['energy_sell_rate'] = self.area_params.pop('energy_rate', None)
                 area.strategy = InfiniteBusStrategy(**self.area_params)
-            else:
-                return False
-            area.activate()
-            area.strategy.event_activate()
-            area.strategy.event_market_cycle()
-        else:
-            area.area_reconfigure_event(**self.area_params)
+                reactivate = True
+            if reactivate:
```

Comment:
I think you have some extra spaces here

Reply:
Sorry, I dont understand. Should I remove indention here?

Reply:
Ah sorry I saw it indented to the right